.github/actions/spelling/expect.txt

```diff
@@ -0,0 +1,339 @@
+aab
+AAFFBB
+abe
+addrpair
+aduit
```

Comment:
this must be a typo, could you please help update this list? I assume it is auto-generated.